project.yaml

```diff
@@ -0,0 +1,4805 @@
+specVersion: 0.0.1
+description: ""
+repository: ""
+schema: ./schema.graphql
+network:
+  endpoint: wss://itn-rpc.polymesh.live
```

Comment:
Do we want this URL environment-based? or different branches may have this different URL.

Reply:
yes, It's something I'm still thinking how to solve exactly